websockets/cookies/META.yml

```diff
@@ -0,0 +1,9 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1141909
```

Comment:
Looks like the right bug, yeah. The bug also mentions 001.html?wss and isn't exactly the same list, but quite close.